src/utils/dump.test.js

```diff
@@ -0,0 +1,15 @@
+import { dump } from './dump';
+
+const result = dump({
+    bytes: new Uint8Array([1, 2, 3]),
+    map: new Map([
+        [new Uint8Array([4, 5, 6]), 'bytes key'],
+        [['a', 'b'], 'array key'],
+        [{c: 'd'}, 'map key'],
+        [new Map([[7, 8]]), 'captial-m map key'],
+        [9, 'number key'],
+        [8_000_000_000n, 'long key'],
+    ]),
+    long: 8_000_000_000n,
+});
+console.log('result', result);
```

Comment:
Should be easy to make this into real test, see

https://github.com/oasisprotocol/oasis-wallet-ext/blob/11d5367ff3795d020aafed4bb8ad75c555466fd3/test/APIService.test.js#L55-L60

Reply:
ooh yeah let me add some of that stuff :laughing:

Reply:
:facepalm: it actually caught a bug where the `n ` prefix was missing on long values